resources_pages/install_ds_stack_mac.md

```diff
@@ -240,46 +248,49 @@ To add the conda-forge channel by typing the following in the terminal:
 conda config --add channels conda-forge
 ```
 
-To install packages individually, we can now use the following command: `conda install <package-name>`. Let's install the key packages needed for the start of our program:
-
-```bash
-conda install \
- jupyterlab=3.* \
- numpy=1.* \
- pandas=1.* \
- flake8=3.* \
- black=21.*
-```
-
+To install packages individually,
+we can now use the following command:
+`conda install <package-name>`.
+After running that command
 `conda` will show you the packages that will be downloaded,
 and you can press enter to proceed with the installation.
 If you want to answer `yes` by default and skip this confirmation step,
 you can replace `conda install` with `conda install -y`.
-
-> **Note:** we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
+Let's try this out in the next section,
+by installing some of the key packages we will use in MDS.
 
 ## JupyterLab setup
 
-We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
+We will be using `JupyterLab` as our main coding environment
+and `pandas` is one of the key data analyses packages in MDS.
+The Jupytext Python package and the JupyterLab git extension facilitates
+using notebooks in JupyterLab together with Git & GitHub.
+The spellchecker helps us correcting typos in our writing
+and the LSP packages fill the same function for our code.
+Install them via the following commands:
 
 ```bash
-conda install nodejs=15.*
```

Comment:
@joelostblom I just noticed that nodejs is not explicitly installed in the new version of the instructions. Is it because it will be installed as a dependency for other packages (e.g. Jupyter Lab)?

Reply:
It is because I removed the only extension that required nodejs. It is only needed when using the old type of extensions for jupyterlab.